content/en/docs/tutorials/trust-policy.md

```diff
@@ -0,0 +1,265 @@
+
+Use `docker buildx build` and `docker push` to build and push a sample image to your registry.
+
+```console
+docker buildx build -t localhost:5000/net-monitor:v1 https://github.com/wabbit-networks/net-monitor.git#main
+docker push localhost:5000/net-monitor:v1
+```
+
+Save the digest value of the image from the output of `docker push`. For example:
+
+```output
+The push refers to repository [localhost:5000/net-monitor]
+2556c54bfdf3: Pushed
+fb6ca4f9c8d3: Pushed
+ded7a220bb05: Pushed
+v1: digest: sha256:1111111111111111111111111111111111111111111111111111111111111111 size: 942
+```
+
+In the above example, the digest value of the image is *sha256:1111111111111111111111111111111111111111111111111111111111111111*.
+
+## Create example certificates and sign the image
+
+Use `notation cert generate-test` to generate three example certificates and trust stores for signing the image. 
+
+```console
+notation cert generate-test valid-example
+notation cert generate-test expired-example
+notation cert generate-test missing-example
+```
+
+- The *valid-example* certificate and trust store will be used to sign the image with a valid signature.
+- The *expired-example* certificate and trust store will be used to sign the image with an expired signature.
+- The *missing-example* certificate and trust store will not be used to sign the image, but will be used to demonstrate the trust policy.
+
+Use `notation ls` to list the current signatures for your image. The following example sets the value of *$IMAGE* to the name of the image and its digest value.
+
+**IMPORTANT**: Always use the digest value of an image when signing since they are immutable. Tag values are mutable and can reference a different container image than the original signed container image.
+
+```console
+IMAGE=localhost:5000/net-monitor@sha256:1111111111111111111111111111111111111111111111111111111111111111
+notation ls $IMAGE
+```
+
+Confirm there are no signatures listed.
+
+Use `notation sign` to sign the image with two different signatures.
+
+```console
+notation sign $IMAGE -k valid-example 
+notation sign $IMAGE --expiry 30s -k expired-example
+```
+
+Use `notation ls` to confirm there are three signatures.
+
+```console
+notation ls $IMAGE
+localhost:5000/net-monitor@sha256:1111111111111111111111111111111111111111111111111111111111111111
+└── application/vnd.cncf.notary.v2.signature
+    ├── sha256:2222222222222222222222222222222222222222222222222222222222222222
+    ├── sha256:3333333333333333333333333333333333333333333333333333333333333333
+```
+
+## Create a trust policy
+
+Create a `trustpolicy.json` with the following trust policy in the notation configuration directory.
+
+**NOTE:** For Linux, the notation configuration directory is `${HOME}/.config/notation/`. For macOS, the notation configuration directory is `${HOME}/Library/Application\ Support/notation/`. For Windows, the notation configuration folder is `%USERPROFILE%\AppData\Roaming\notation\`.
```

Comment:
I noticed there is `\ ` in the path `${HOME}/Library/Application\ Support/notation/`. It is correct?

Reply:
@yizha1 yes,  due to the space in the path you need an escape character